src/EscolaLmsServiceProvider.php

```diff
@@ -3,24 +3,17 @@
 namespace EscolaLms\Core;
 
 use EscolaLms\Core\Providers\Injectable;
-use EscolaLms\Core\Repositories\AttachmentRepository;
 use EscolaLms\Core\Repositories\ConfigRepository;
-use EscolaLms\Core\Repositories\Contracts\AttachmentRepositoryContract;
 use EscolaLms\Core\Repositories\Contracts\ConfigRepositoryContract;
-use EscolaLms\Core\Services\AttachmentService;
 use EscolaLms\Core\Services\ConfigService;
-use EscolaLms\Core\Services\Contracts\AttachmentServiceContract;
 use EscolaLms\Core\Services\Contracts\ConfigServiceContract;
-use Illuminate\Support\Facades\Blade;
 use Illuminate\Support\ServiceProvider;
 
 class EscolaLmsServiceProvider extends ServiceProvider
 {
     use Injectable;
```

Comment:
imho this is https://github.com/EscolaLMS/Core/issues/9

Reply:
There are probably many useless things in this package, check TODO's I added in comments in this PR...